mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -72,6 +71,11 @@ def __init__(
             timestamp_precision
         )
 
+        self.database = kwargs.get(
```

Comment:
why do we need the capability to provide custom database?
I saw this is used in test-2, but i'm not sure what does test-2 tests that is relevant to this bug?